easybuild/easyblocks/o/openmpi.py

```diff
@@ -160,11 +160,12 @@ def load_module(self, *args, **kwargs):
     def sanity_check_step(self):
         """Custom sanity check for OpenMPI."""
 
-        bin_names = ['mpicc', 'mpicxx', 'mpif90', 'mpifort', 'mpirun', 'ompi_info', 'opal_wrapper']
+        bin_names = ['mpicc', 'mpicxx', 'mpif90', 'mpifort', 'ompi_info', 'opal_wrapper']
         if LooseVersion(self.version) >= LooseVersion('5.0.0'):
             bin_names.append('prterun')
         else:
-            bin_names.append('orterun')
+            if '--with-orte=no' not in self.cfg['configopts'] and '--without-orte' not in self.cfg['configopts']:
```

Comment:
Can you clarify this a bit? Does configuring without `orte` imply not having `mpirun` at all?

Reply:
Exactly. `mpirun` is not built. `orterun` is built, but at least IMO not really used and probably should not be built either.

Reply:
without orterun for older versions you can't start openmpi code. It's always used.